### PR TITLE
Fixing metadata comparison issue with custom axes

### PR DESCRIPTION
**Pull request by kormic911**
This should fix the issue with damaged custom axes no longer being registered properly, switched to doing a string compare on the item name vs. the item stack check which also included the metadata.

Also switched to spaces not tabs! :)

This is in reference to issue #49 also

### CODE REVIEW

File: src/main/java/gigaherz/survivalist/ConfigManager.java [hunk 1]
```diff
@@ -254,8 +254,9 @@ public int getAxeLevel(ItemStack stack)
     {
         for (Pair<ItemStack, Integer> customAxe : customChoppingAxes)
         {
-            if (ItemStack.areItemsEqual(customAxe.getLeft(), stack))
+            if (customAxe.getLeft().getUnlocalizedName().equalsIgnoreCase(stack.getUnlocalizedName())) {
```
Comment (gigaherz) [inline src/main/java/gigaherz/survivalist/ConfigManager.java]: That's NOT the way to compare. I'm not at home, but I believe there's a function like ItemStack.areItemsEqualIgnoreDurability

Comment (kormic911) [inline src/main/java/gigaherz/survivalist/ConfigManager.java]: You're right, I totally overlooked that method!

**Comment by kormic911** [comment]
I changed it to ItemStack.areItemsEqualIgnoreDurability and verified it does work properly :)

**Comment by gigaherz** [comment]
I'll do a bugfix release later today (UTC+2 here), unless I get home very late, in which case it will be tomorrow.

**Comment by kormic911** [comment]
Np, love the mod! Always happy to help out :)

**Pull request merged**